apis/management.cattle.io/v3/authn_types.go

```diff
@@ -111,11 +111,36 @@ type GithubConfigTestOutput struct {
 }
 
 type GithubConfigApplyInput struct {
-	GithubConfig GithubConfig `json:"githubConfig, omitempty"`
+	GithubConfig GithubConfig `json:"githubConfig,omitempty"`
 	Code         string       `json:"code,omitempty"`
 	Enabled      bool         `json:"enabled,omitempty"`
 }
 
+type AzureADConfig struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	AuthConfig        `json:",inline" mapstructure:",squash"`
+
+	AzureADEndpoint      string `json:"azureADEndpoint,omitempty" norman:"default=https://login.microsoftonline.com/,required,notnullable"`
```

Comment:
it is redundant to have AzureAD in the name of the fields. can you drop that from these?